### PR TITLE
Review repo

### PR DESCRIPTION
**Pull request by AnelisaDee**
This is a YouTube page clone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a youtube clone.
```
Comment (AnelisaDee) [inline README.md:1]: KIndly make sure to clone the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your repo in order to make the repo looks more professional and try to customize it according to your project.

**Make sure your README looks more professional by defining the headers of each section. Kindly follow the provided template to achieve it**

File: index.html [hunk 2]
```diff
@@ -4,9 +4,103 @@
...
+              <button class="search"><i class="material-icons">search</i></button>
+            </form>
+            <i class="material-icons mic">mic</i>
+          </div>
+          <div class="icons right">
+            <i class="material-icons">videocam</i>
+            <i class="material-icons">apps</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons display-this">account_circle</i>
+          </div>
```
Comment (AnelisaDee) [inline index.html:29]: Please add a link to all your icons. It represent a generic navigation section of the document, please use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) for more info.

**Review by AnelisaDee — COMMENTED** [review]
**Highlight**
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Good commit history ✔️ 
 
**Changes Required** ♻️ 
Please check my comments below and make all the required changes.

Submit review when you are done.
Thanks 😁

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @AnelisaDee    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**